storage/ndb/rest-server/data-access-rondb/src/db-operations/pk/pkr-operation.cpp

```diff
@@ -351,3 +357,22 @@ RS_Status PKROperation::Abort() {
 
   return RS_OK;
 }
+
+RS_Status PKROperation::HandleNDBError(RS_Status status) {
```

Comment:
This is doing something very specific - refresh state if incorrect schema. You could name the function accordingly.